.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
@ericdao-ttt I'm glad that you added prettier to help formatting the code. However, it is not related to the TypeScript definition, and it caused a lot of code differences without importing any functionality change. Please consider move this to a separate branch.